pkgs/tools/virtualization/google-compute-engine/default.nix

```diff
@@ -25,7 +25,7 @@ buildPythonApplication rec {
     for file in $(find google_compute_engine -type f); do
       substituteInPlace "$file" \
         --replace /bin/systemctl "${systemd}/bin/systemctl" \
-        --replace /bin/bash "${bash}/bin/bash" \
+        --replace /bin/bash "${bashInteractive}/bin/bash" \
```

Comment:
you also replace this in the header of this file.

Reply:
Sorry, I forget to push my local changes...